scenarios/mongoose_simple_docker.erl

```diff
@@ -163,21 +163,12 @@ make_jid(Id) ->
 
 -spec pick_server() -> binary().
 pick_server() ->
-    Servers = env_servers(),
+    Servers = amoc_config:get('XMPP_SERVERS'),
     S = size(Servers),
     N = erlang:phash2(self(), S) + 1,
     element(N, Servers).
 
--spec env_servers() -> {binary()}.
-env_servers() ->
-    List = re:split(os:getenv("AMOC_XMPP_SERVERS"), "\s",
-                    [{return, binary}, trim]),
-    list_to_tuple(List).
-
 -spec set_env_interarrival() -> ok.
 set_env_interarrival() ->
-    Interarrival = case os:getenv("AMOC_INTERARRIVAL") of
-                       false -> 100;
-                       List -> list_to_integer(List)
-                   end,
+    Interarrival = amoc_config:get('INTERARRIVAL'),
     application:set_env(amoc, interarrival, Interarrival).
```

Comment:
This is bad, because we break the abstraction and set `interarrival` instead of having it in the amoc_config:get/1 directly by passing `AMOC_interarrival` instead of `AMOC_INTERARRIVAL`. It has to be fixed.